src/components/contentful/contentfulOpenLetterForm/contentfulOpenLetterForm.js

```diff
@@ -14,16 +13,16 @@ const axiosConfig = {
 
 const OpenLetterForm = ({ content }) => {
   const {
-    writeALetterButtonText,
-    readYourLetterButtonText,
+    /* writeALetterButtonText,
+    readYourLetterButtonText, */
     title,
     description,
     defaultLanguage,
   } = content;
 
   const [expandOpenLetterStart, setExpandOpenLetterStart] = useState(false);
   const [expandOpenLetterRead, setExpandOpenLetterRead] = useState(false);
-  const [accessKey, setAccessKey] = useState('');
+  const [letterKey, setLetterKey] = useState('');
```

Comment:
Is there a reason to use `letterKey` instead of `accessKey`? I'm also using `accessKey` in backend and the API, so using `accessKey` would keep things more consistent.

As an after thought if we really want to go with better naming, then I guess `accountId` would make more sense, since key and password are both kind of similar. In that case, it'll also be nice to update the service backend and database column names to keep everything consistent.

I'm ok with `letterKey` though, if you'd like to keep it : )

Reply:
Yeah, there is: "Also, I renamed the accessKey-attribute, because there is an [HTML-attribute called accesskey](https://developer.cdn.mozilla.net/en-US/docs/Web/HTML/Global_attributes/accesskey), so it's sort of reserved name." It caused some linter errors, and I thought it would be best to refactor from the front end :) 

`accountId` would be good too! I honestly wasn't sure about the logic (did not look into the backend  code 😅 ) so I went with the key idea to be sure 😄